c-cannons/bangazon-initial_site
# Ml product status

#### Descriptions

Added and inventory and sold attribute to each product card

#### Steps to Test
```
git fetch origin ml-product-status
git checkout ml-product-status
```
1. Go to my products and make sure inventory: and sold: are showing correctly.

#### Link to Feature Ticket

#15
